sqlglot/dialects/prql.py

```diff
@@ -103,6 +104,18 @@ def _parse_take(self, query: exp.Query) -> t.Optional[exp.Query]:
             num = self._parse_number()  # TODO: TAKE for ranges a..b
             return query.limit(num) if num else None
 
+        def _parse_order_by(self, query: exp.Select) -> t.Optional[exp.Query]:
+            order = self._parse_sort(exp.Order, TokenType.L_BRACE)
+            if order and not self._match(TokenType.R_BRACE, expression=query):
+                self.raise_error("Expecting }")
+            return (
+                query.order_by(order, copy=False)
+                if order
+                else query.order_by(
+                    self.expression(exp.Order, expressions=self._parse_csv(self._parse_ordered))
+                )
+            )
```

Comment:
I think this section needs to be:

```python
l_brace = self._match(TokenType.L_BRACE)
expressions = self._parse_csv(self._parse_ordered)
if l_brace and not self._match(TokenType.R_BRACE):
    self.raise_error("Expecting }")
...
```

AFAICT we can only omit the braces for the `sort` operator only when there's a single column, so this should cover both that and the multiple column case.

Reply:
We'll probably need to override `_parse_ordered` to convert this syntax into nodes with `ASC` / `DESC`:

```
sort {age, -tenure, +salary}
```